2-mandatory/2-financial-times.js

```diff
@@ -14,24 +14,47 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let smallTitle = allArticleTitles[0];
+    for(let item = 1; item < allArticleTitles.length; item++){
+        if(allArticleTitles[item].length < smallTitle.length) {
+            smallTitle = allArticleTitles[item];
+        }
+    }
+    return smallTitle;
+
+
 }
 
+
 /*
     The editor of the FT has realised that headlines which have numbers in them get more clicks!
     Implement the function below to return a new array containing all the headlines which contain a number.
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    /*const hasNumbers = allArticleTitles.filter(item => /\d/.test(item) );
+    return hasNumbers;*/
+    const numberTitle = [];
+    for(let title of allArticleTitles){
+        for(let index = 0; index < title.length; index++){
+            if((title[index]>= '0') && (title[index]<='9')){
+                numberTitle.push(title);
+                break;
+            }
+        }
+    }
```

Comment:
Well done @munozirianni1988 Your code is very good. I like the use of the break to stop the loop from continuing once a number is found, very efficient. Remember to delete any commented code though when submitting :).